pkg/gds/emails/sentlogs.go

```diff
@@ -0,0 +1,37 @@
+// This contains helper functions to help confirm if emails were sent.
+package emails
+
+import (
+	"time"
+
+	"github.com/trisacrypto/directory/pkg/gds/models/v1"
+	pb "github.com/trisacrypto/trisa/pkg/trisa/gds/models/v1beta1"
+)
+
+// VerifyEmailSentVASPContact checks if an email has been sent to the VASP contacts in some time window (e.g. the past 30 days).
+// Returns false for either email has not been sent or an error occurred.
+func VerifyEmailSentVASPContact(vasp *pb.VASP, kind string, timeWindowDays int) (status bool, err error) {
```

Comment:
Instead of a bool, can we return a count of the sent emails and also require a reason string to be passed in? I realize this is a bit of a refactor but @DanielSollis will need to check how many emails are sent since both the 30 day and 7 days reminders are classified as the same reason. We can rename this method to `GetSentEmailCount` or something.